src/redist/targets/packaging/windows/clisdk/bundle.wxs

```diff
@@ -14,6 +14,10 @@
         WixBundleInstalled OR ((NOT (DOTNETHOME_X64 ~= DOTNETHOME_X86)) OR DOTNETHOMESIMILARITYCHECKOVERRIDE)
     </bal:Condition>
 
+    <bal:Condition Message="The installation path for ARM64 SDK installations: &quot;[DOTNETHOME_ARM64]&quot; cannot be the same as for x86 SDK installations: &quot;[DOTNETHOME_X86]&quot;">
```

Comment:
I wonder if this is necessary. x86/x64 can be installed SxS on the same machine, but I don't see how we'd support installing x86 SxS with arm64.

Reply:
A lot of the ARM64 ecosystem in Windows relies on emulation, which is x86. Hence I tried to leave the support in this.